src/Installer/redist-installer/targets/GenerateMSIs.targets

```diff
@@ -405,6 +409,27 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="GenerateRuntimeAnalyzersNupkg"
+          DependsOnTargets="GenerateLayout;MsiTargetsSetupInputOutputs"
+          Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' And '$(DotNetBuild)' != 'true' "
```

Comment:
@jjonescz I just saw this PR. Why did you add the `DotNetBuild` condition here? Shouldn't the package get produced in the product build? Note that starting with P4 we will ship the product from the VMR and this asset will be missing from the product then.

Reply:
Thanks, I will look into removing the condition.